resolvers/event.js

```diff
@@ -9,6 +9,53 @@ const eventResolvers = {
   },
   Event: {
     createdBy: (obj, args, context, info) => db.user.findByPk(obj.createdBy)
+  },
+  Mutation: {
+    addEvent: async (parent, args) => {
+      // Check if createdBy is valid
+      const user = await db.user.findByPk(args.createdBy);
+      if (!user) {
+        throw new Error("Invalid user ID");
+      }
+      return db.event.create({
+        name: args.name,
+        eventDate: args.eventDate,
+        createdBy: args.createdBy,
+        isActive: args.isActive
+      });
+    },
+    updateEvent: async (parent, args) => {
+      const event = await db.event.findByPk(args.id);
+      const user = args.createdBy
+        ? await db.user.findByPk(args.createdBy)
+        : true;
+      if (!(event && user)) {
+        if (!event) throw new Error("Invalid event ID");
+        else throw new Error("Invalid user ID");
+      }
```

Comment:
seems excessively complicated
```suggestion
      if (!event) {
        throw new Error("Invalid event ID");
      } else if (!user) {
        throw new Error("Invalid user ID");
      }
```